### PR TITLE
Annotation Tool Visibility Button

### PR DESCRIPTION
**Pull request by PaulHax**
Adds an eyeball button in the MeasurmentToolList to toggle `hidden` state of annotation tools. 
image

Also adds "Hide" option to right click context menu on annotations.

### CODE REVIEW

File: src/components/MeasurementsToolList.vue [hunk 1]
```diff
@@ -39,59 +39,80 @@ const remove = (id: ToolID) => {
...
+            <v-icon v-if="tool.hidden">mdi-eye-off</v-icon>
+            <v-icon v-else>mdi-eye</v-icon>
+            <v-tooltip location="top" activator="parent">{{
+              tool.hidden ? 'Show' : 'Hide'
+            }}</v-tooltip>
+          </v-btn>
+          <v-btn icon variant="text" @click="jumpTo(tool.id)">
+            <v-icon>mdi-target</v-icon>
+            <v-tooltip location="top" activator="parent"
+              >Reveal Slice</v-tooltip
```
Comment (floryst) [inline src/components/MeasurementsToolList.vue:71]: Got some formatting snafu here. I'm still not sure why prettier does this.

File: src/composables/annotationTool.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        } satisfies ContextMenuEvent);
+      }
+    });
+  });
+
+  onBeforeUnmount(() => {
+    if (rightClickSub) {
+      rightClickSub.unsubscribe();
+      rightClickSub = null;
+    }
```
Comment (floryst) [inline src/composables/annotationTool.ts:98]:
```suggestion
    rightClickSub?.unsubscribe();
    rightClickSub = null;
```

File: src/composables/annotationTool.ts [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      );
+    });
+  });
+
+// --- Context Menu --- //
+
+export const useContextMenu = <ToolID extends string>() => {
+  // open function typing workaround until this fixed
+  // https://github.com/vuejs/core/issues/8373
+  const contextMenu = ref<
```
Comment (floryst) [inline src/composables/annotationTool.ts:58]: I think this just be typed as `{ open: ... }`. Interface-wise, we only need `open` and we don't care about having an entire component's interface.

File: src/composables/annotationTool.ts (hunk 2 above)
Comment (PaulHax) [reply src/composables/annotationTool.ts:98]: 🧹

File: src/components/MeasurementsToolList.vue (hunk 1 above)
Comment (PaulHax) [reply src/components/MeasurementsToolList.vue:71]: 👍️ 
🧓 Lines don't grow on trees sonny.

File: src/composables/annotationTool.ts (hunk 3 above)
Comment (PaulHax) [reply src/composables/annotationTool.ts:58]: ✅

**Review by floryst — APPROVED** [review]

**Pull request merged**